backend/app/main.py

```
@@ -49,8 +86,31 @@ def create_app() -> FastAPI:
     async def health_check():
-        """Health check endpoint."""
-        return {"status": "ok", "version": "2.0.0"}
+        """Health check endpoint with database connectivity test."""
+        try:
+            # Test database connection
+            from common.db.connection import SessionLocal
+            db = SessionLocal()
+            try:
+                db.execute(text("SELECT 1"))
+                db_status = "connected"
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)